qa/L0_multi_model_profile/check_results.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2022, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import defaultdict
+import argparse
+import sys
+import yaml
+import os
+
+
+class TestOutputValidator:
+    """
+    Functions that validate the output
+    of the test
+    """
+
+    def __init__(self, config, test_name, analyzer_log):
+        self._config = config
+        self._models = config['profile_models']
+        self._analyzer_log = analyzer_log
+
+        check_function = self.__getattribute__(f'check_{test_name}')
+
+        if check_function():
+            sys.exit(0)
+        else:
+            sys.exit(1)
+
+    def check_profile_logs(self):
+        """
+        Check that each model was profiled the number of times
+        corresponding with batch size and concurrency combinations
+
+        For multi-model with 2 models, the expected number of measurements
+        is X*Y, where:
+        - X is the number of measurements that we would expected 
+        for model 1 in a single-model case
+        - Y is the number of measurements that we would expected 
+        for model 1 in a single-model case
```

Comment:
this should be model 2 (or you can change this to model A/B which is my preference: models get letters, configs get numbers)

Reply:
fixed